Source/Core/DolphinQt2/CMakeLists.txt

```diff
@@ -120,19 +120,20 @@ set(SRCS
   TAS/StickWidget.cpp
   TAS/IRWidget.cpp
   Updater.cpp
+  ${UI_HEADERS}
```

Comment:
Where is UI_HEADERS defined? I can't find it :/

Reply:
Looks like it's just unused because we don't have any calls to `qt5_wrap_ui`.

Reply:
Removed it